pepper-apis/src/main/java/org/broadinstitute/ddp/model/study/ActivityMapping.java

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.ddp.model.study;
```

Comment:
I think this belongs in the `ddp.model.study` package instead of `ddp.model.dsm`. It's now more of a "study configuration" concept than a DSM concept.